x-pack/plugins/apm/public/components/app/service_map/popover/service_contents.tsx

```diff
@@ -46,7 +46,12 @@ export function ServiceContents({
   const { query } = useApmParams('/*');
 
   if (
-    !('rangeFrom' in query && 'rangeTo' in query) ||
+    !(
+      'rangeFrom' in query &&
+      'rangeTo' in query &&
+      'comparisonEnabled' in query &&
+      'offset' in query
+    ) ||
     !query.rangeFrom ||
     !query.rangeTo
   ) {
```

Comment:
Is `comparisonEnabled` and `offset` now required? If so, the error message should indicate that (it only mentions `rangeFrom` and `rangeTo`.

Reply:
This check is not needed here, removed in https://github.com/elastic/kibana/pull/130152/commits/7c9fb7feaf3896b4d770a460f04bce0918195605